test_data/rules/unnecessary_parenthesis.dart

```diff
@@ -145,8 +145,25 @@ main() async {
   a2 = (1 == 1); // OK
   a2 = (1 == 1) || "".isEmpty; // OK
   var a3 = (1 + 1); // LINT
+
+  // Tests for Literal and PrefixedIdentifier
+  var a4 = (''); // LINT
```

Comment:
You mention only false positives but these tests validate (new) reports. Is that right? Is this actually fixing false negatives?

Reply:
Oh sorry, I mixed up the two. I am only fixing false negatives here. I.e. only adding more complaint cases.